README.md

```diff
@@ -1,27 +1,23 @@
 # libsignal-service-rs
 
-[![Build Status](https://github.com/whisperfish/libsignal-service-rs/workflows/CI/badge.svg?branch=main)](https://github.com/whisperfish/libsignal-service-rs/actions/workflows/ci.yaml?query=branch:main)
-[![API Docs](https://img.shields.io/badge/docs-libsignal--service-blue)](https://whisperfish.github.io/libsignal-service-rs/libsignal_service)
+[![Build Status](https://github.com/whisperfish/libsignal-service-rs/workflows/CI/badge.svg?branch=main)][ci_workflow]
+[![API Docs](https://img.shields.io/badge/docs-libsignal--service-blue)][rustdocs]
 
-A Rust version of the [libsignal-service-java][lsj] library for communicating
-with Signal servers.
+A Rust version of the [libsignal-service-java][lsj] library which implements the core functionnality to communicate with [Signal][signal] servers. It is based on the official Rust version of [libsignal][lsg]
```

Comment:
```suggestion
A Rust version of the [libsignal-service-java][lsj] library which implements the core functionality to communicate with [Signal][signal] servers. It is based on the official Rust version of [libsignal][lsg]
```

Since we're diverging quite a bit from lsj, we might just say that the library implements the Signal API surface for building Signal clients?